### PR TITLE
Fixed support for non-ASCII MusicBrainz usernames

### PR DESCRIPTION
**Pull request by gentlecat**
```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/vagrant/webserver/views/index.py", line 12, in index
    last_submitted_data=get_last_submitted_recordings())
  File "/usr/local/lib/python2.7/dist-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/usr/local/lib/python2.7/dist-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/vagrant/webserver/templates/index/index.html", line 1, in top-level template code
    {%- extends 'base.html' -%}
  File "/vagrant/webserver/templates/base.html", line 28, in top-level template code
    {%- include 'navbar.html' -%}
  File "/vagrant/webserver/templates/navbar.html", line 36, in top-level template code
    {{ current_user.musicbrainz_id }} <span class="caret"></span>
UnicodeDecodeError: 'ascii' codec can't decode byte 0xd0 in position 0: ordinal not in range(128)
```
:bug:

### CODE REVIEW

File: db/stats.py [hunk 1]
```diff
@@ -20,8 +20,7 @@ def get_last_submitted_recordings():
                               OFFSET 10""")
             last_submitted_data = cursor.fetchall()
         last_submitted_data = [
-            (r[0], r[1].decode("UTF-8"), r[2].decode("UTF-8"))
```
Comment (gentlecat) [inline db/stats.py]: This is no longer needed.

**Comment by gentlecat** [comment]
*Tests actually pass, it's just coverage plugin being crazy in Jenkins. http://ci.roman.zone/job/acousticbrainz-server/83/console*

**Comment by alastair** [comment]
cool, good fix. This is the recommended way of dealing with unicode in python - do the conversion from bytes to unicode at the border.

**Comment by gentlecat** [comment]
:shipit:?

**Comment by alastair** [comment]
yes

**Comment by gentlecat** [comment]
:sparkles: :ship: :sparkles:

**Pull request merged**